rerun-failed-run/rerun-failed-run.yml

```diff
@@ -0,0 +1,73 @@
+name: Retrigger failed run action
```

Comment:
As this is a new action, it also needs a `README.md` to explain how it works and how to use it.